src/js/_enqueues/vendor/tinymce/plugins/wordpress/plugin.js

```diff
@@ -147,7 +147,6 @@ tinymce.PluginManager.add( 'wordpress', function( editor ) {
 						'class="mce-object" ' +
```

Comment:
Thanks for starting this PR.

To make a visual distinction between the style and script placeholders, you could:

1. Add small `style.png` and `script.png` images to the `src\js\_enqueues\vendor\tinymce\skins\wordpress\images` directory.
2. Add styles to `src\js\_enqueues\vendor\tinymce\skins\wordpress\wp-content.css`, probably after `.mce-content-body img[data-wp-more="nextpage"]`.
```
.mce-content-body img.mce-object-style {
	background-image: url( images/style.png );
}

.mce-content-body img.mce-object-script {
	background-image: url( images/script.png );
}
```
3. Add an extra class to the placeholder images.
```suggestion
						'class="mce-object mce-object-' + tag + '" ' +
```
----

I made these simple images with "CSS" and "JS" (in Trebuchet MS font) at 20 by 20 pixels, if you would like to use them.
- style.png ![style placeholder](https://github.com/user-attachments/assets/74967b62-ee15-4c9f-b140-fa4754826bdc)
- script.png ![script placeholder](https://github.com/user-attachments/assets/28f25da9-9044-47cb-8b44-0ab7b6309aa5)

![placeholders side-by-side](https://github.com/user-attachments/assets/92494595-c70e-43af-8815-4cc78756afcc)

Reply:
Or a similar design in vector format:
- style.svg ![CSS](https://github.com/user-attachments/assets/a6a48cbc-5b5a-4b64-8e7b-a04b2fc136d2)
- script.svg ![JS](https://github.com/user-attachments/assets/b6a1abec-d7eb-4d58-bcc9-a7479c046c53)

```
.mce-object-style {
	background-image: url( images/style.svg );
}

.mce-object-script {
	background-image: url( images/script.svg );
}
```

Reply:
Thanks @sabernhardt for the suggestions. 

I have used SVGs for the images and addressed all the changes in the latest commit.